.vimrc

```diff
@@ -0,0 +1,39 @@
+" use 2-space soft tabs.  Some of these may be redundant
+:set softtabstop=2
+:set shiftwidth=2
+:set tabstop=2
+:set expandtab
+" turn on cursor column and row listing
+:set ruler
+" configure window status info
+:set statusline=%<%F\ %n%h%m%r%=%-14.(%l,%c%V%)\ %P
+:set laststatus=2
+" configure spellcheck, but turn it off by default: enable with ':set spell'
+:setlocal spell spelllang=en_us
+:set spellfile=~/.vim.myspell.add
+:set nospell
+" configure display of hidden characters: these can be modfied as desired, or
+" turned off with ':set listchars='
+:set listchars=eol:$,tab:>-,trail:~,extends:>,precedes:<
```

Comment:
Hi Dave,  you mentioned that the dollar-sign EOL marker can be unintentionally cut-and-pasted by users.
Another approach is to highlight trailing white space using color and avoid the EOL character.
This is what our .emacs config does, and it is also possible in vim:
:highlight ExtraWhitespace ctermbg=red guibg=red
" The following alternative may be less obtrusive.
:highlight ExtraWhitespace ctermbg=darkgreen guibg=lightgreen
" Try the following if your GUI uses a dark background.
:highlight ExtraWhitespace ctermbg=darkgreen guibg=darkgreen

this is from https://vim.fandom.com/wiki/Highlight_unwanted_spaces

Reply:
I tried :highlight ExtraWhitespace ... with no success.  'help highlight' also doesn't list ExtraWhitespace as a valid group.

Reply:
Using
:autocmd ColorScheme * highlight ExtraWhitespace ctermbg=red guibg=red
works for me